quote.c

```diff
@@ -47,7 +47,16 @@ void sq_quote_buf_pretty(struct strbuf *dst, const char *src)
 {
 	static const char ok_punct[] = "+,-./:=@_^";
 	const char *p;
-
+	
+	/* 
+	In case of null or empty strings, add a '' to ensure we 
+	don't inadvertently drop these tokens
+	*/
```

Comment:
super-nit:  they like comments like this to look like

    /*
     * In case ...
     * don't ...
     */